### DISCUSSION TITLE
How to generate a store with the latest version of @storefront-ui/vue?

### DISCUSSION THREAD

**Discussion by Tofandel** [DivanteLtd/vue-storefront#6994] [Q&A] [answered]
Running `npx @vue-storefront/cli generate store` creates a project with an outdated version of `@storefront-ui/vue` (0.11.4 vs latest 2.6.0)

Is there a way to create a store with the latest version instead of having to go through the very painful manual upgrade route for a new project?

I mean this is litterally 2 major versions with a lot of breaking changes... https://docs.storefrontui.io/v2/vue/migration.html#migration-guide

Am I missing something obvious here?

I really don't want to start a project on something that will need to be rewritten from the get go

**Comment by Tofandel** [comment, marked as answer]
Seems there is a boilerplate instead of the cli

https://github.com/vuestorefront/storefront-nuxt3-boilerplate/tree/develop

The doc really needs to be updated, it's a mix of 2 versions right now

**Reply by ithuis** [reply]
@Tofandel same here... it's all really confusing.  did you start with the boilerplate?

**Reply by Tofandel** [reply]
Yes I did, seems all is fine with it, but the doc for it is incomplete
